docker-compose.yml

```diff
@@ -124,9 +135,20 @@ services:
       KAFKA_SSL_CLIENT_AUTH: "required"
       KAFKA_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM: "HTTPS"
       KAFKA_ZOOKEEPER_SET_ACL: "true"
+      # Schema Validation
+      KAFKA_CONFLUENT_SCHEMA_REGISTRY_URL: "https://schemaregistry:8085"
+      # Until 5.4 GA (https://github.com/confluentinc/schema-registry/pull/957): use KAFKA_OPTS
+      #KAFKA_CONFLUENT_SCHEMA_REGISTRY_SSL_TRUSTSTORE_LOCATION: "/etc/kafka/secrets/kafka.kafka1.truststore.jks"
+      #KAFKA_CONFLUENT_SCHEMA_REGISTRY_SSL_TRUSTSTORE_PASSWORD: "confluent"
+      #KAFKA_CONFLUENT_SCHEMA_REGISTRY_SSL_KEYSTORE_LOCATION: "/etc/kafka/secrets/kafka.kafka1.keystore.jks"
+      #KAFKA_CONFLUENT_SCHEMA_REGISTRY_SSL_KEYSTORE_PASSWORD: "confluent"
       KAFKA_OPTS: -Djava.security.auth.login.config=/etc/kafka/secrets/broker_jaas.conf
+                  -Djavax.net.ssl.trustStore=/etc/kafka/secrets/kafka.kafka2.truststore.jks
+                  -Djavax.net.ssl.trustStorePassword=confluent
+                  -Djavax.net.ssl.keyStore=/etc/kafka/secrets/kafka.kafka2.keystore.jks
+                  -Djavax.net.ssl.keyStorePassword=confluent
   connect:
-    image: confluentinc/cp-kafka-connect:5.4.0-SNAPSHOT
+    image: confluentinc/cp-kafka-connect:5.3.1
```

Comment:
I'm curious why you're testing/running 5.4 beta cp-server with the other tools at 5.3.1?

Reply:
I forget which one (SR or C3 perhaps?) but some of the 5.4 beta images weren't available yet for other components.